instrumentation/restlet/restlet-1.0/library/src/test/groovy/io/opententelemetry/instrumentation/restlet/v1_0/spring/AbstractSpringTest.groovy

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opententelemetry.instrumentation.restlet.v1_0.spring
+
+import com.noelios.restlet.StatusFilter
+import io.opentelemetry.instrumentation.restlet.v1_0.RestletTracing
+import io.opentelemetry.instrumentation.restlet.v1_0.spring.AbstractSpringServerTest
+import io.opentelemetry.instrumentation.test.LibraryTestTrait
+import org.restlet.Restlet
+import org.restlet.Route
+import org.restlet.Router
+import org.restlet.util.RouteList
+
+abstract class AbstractSpringTest extends AbstractSpringServerTest implements LibraryTestTrait {
```

Comment:
Nit: WDYT about renaming it to `AbstractSpringServerLibraryTest`? Right now it looks a bit strange that `AbstractSpringTest` extends `AbstractSpringServerTest`, but if you mention that it's for library instrumentation then it'll be perfectly fine.

Reply:
The current naming is indeed a bit weird, I'll change it to `AbstractSpringServerLibraryTest`. Thanks!